plans/admin.py

```diff
@@ -115,58 +115,58 @@ class InvoiceInline(admin.TabularInline):
     model = Invoice
     extra = 0
     raw_id_fields = (
-        'user',
+        'customer',
     )
 
 
 class OrderAdmin(admin.ModelAdmin):
     list_filter = ('status', 'created', 'completed', 'plan__name', 'pricing')
-    raw_id_fields = ('user',)
+    raw_id_fields = ('customer',)
     search_fields = (
-        'id', 'user__username', 'user__email', 'invoice__full_number'
+        'id', 'customer__email', 'invoice__full_number'
     )
     list_display = (
-        'id', 'name', 'created', 'user', 'status', 'completed',
+        'id', 'name', 'created', 'customer', 'status', 'completed',
         'tax', 'amount', 'currency', 'plan', 'pricing',
         'plan_extended_from', 'plan_extended_until',
     )
     list_display_links = list_display
     actions = [make_order_completed, make_order_invoice]
-    inlines = (InvoiceInline, )
+    inlines = (InvoiceInline,)
 
     def queryset(self, request):
-        return super(OrderAdmin, self).queryset(request).select_related('plan', 'pricing', 'user')
+        return super(OrderAdmin, self).queryset(request).select_related("plan", 'pricing', 'customer')
 
 
 class InvoiceAdmin(admin.ModelAdmin):
     search_fields = (
         'full_number', 'buyer_tax_number',
-        'user__username', 'user__email'
+        'customer__email'
```

Comment:
This breaks what might be current users used to. Surely - this breaks your site where your model is missing the `username` field. But also after this modification some users might be missing `email` field in their models and it would not work for them.

I would prefer to make configuration and/or autodetection of missing fields out from this.